Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -564,8 +564,9 @@ open class TestStore<State, Action, ScopedState, ScopedAction, Environment> {
   ///   - initialState: The state the feature starts in.
   ///   - reducer: The reducer that powers the runtime of the feature.
   public init<Reducer: ReducerProtocol>(
-    initialState: State,
+    initialState: @autoclosure () -> State,
     reducer: Reducer,
+    updateDependencies: (inout DependencyValues) -> Void = { _ in },
```

Comment:
I know this'll be a trailing closure without argument labels most of the time, but what do you think of:
```suggestion
    prepareDependencies: (inout DependencyValues) -> Void = { _ in },
```

Reply:
yep that works for me. will update